### DISCUSSION TITLE
BeagleBoneBlack ArchLinuxArm Build Failure

### DISCUSSION THREAD

**Discussion by CastleCaptain** [jcorporation/myMPD#720] [Q&A]
I have tried building the master and devel versions of the 9.2.3 release but it fails to build at 93% each time with the same error.   A screen cap of the debug run error is this: 
[ 93%] Building C object CMakeFiles/mympd.dir/src/lib/cert.c.o
/usr/bin/cc  -I/home/myMPD/debug -I/home/myMPD -I/home/myMPD/dist/libmpdclient/include -fstack-protector-strong -D_FORTIFY_SOURCE=2   -ffunction-sections -fdata-sections -Wall -Werror -Wvla -Winit-self -Wuninitialized       -Werror=array-bounds -Wextra -pedantic -Wunused-parameter -Wshadow -Wwrite-strings  -Wformat=2 -Wstrict-prototypes -Wold-style-definition -Wredundant-decls -Wnested-externs        -Wmissing-include-dirs -Wundef -Wformat-nonliteral -Wsign-compare -Wconversion   -Wno-stringop-overread -Wno-array-bounds   -fsanitize=address     -fsanitize=undefined -fsanitize=shift -fsanitize=integer-divide-by-zero -fsanitize=unreachable -fsanitize=vla-bound     -fsanitize=null -fsanitize=return -fsanitize=signed-integer-overflow -fsanitize=bounds -fsanitize=bounds-strict     -fsanitize=alignment -fsanitize=object-size -fsanitize=float-divide-by-zero -fsanitize=float-cast-overflow     -fsanitize=nonnull-attribute -fsanitize=returns-nonnull-attribute -fsanitize=bool -fsanitize=enum -fsanitize=vptr     -static-libasan -DMG_ENABLE_OPENSSL=1 -DMG_ENABLE_DIRECTORY_LISTING=1 -DMG_ENABLE_SSI=0 -DMG_ENABLE_IPV6=1 -D MJSON_ENABLE_PRINT=0 -D MJSON_ENABLE_BASE64=0 -D MJSON_ENABLE_RPC=0   -D MJSON_ENABLE_PRETTY=0 -D MJSON_ENABLE_MERGE=0 -ggdb -Og -g -MD -MT CMakeFiles/mympd.dir/src/lib/cert.c.o -MF CMakeFiles/mympd.dir/src/lib/cert.c.o.d -o CMakeFiles/mympd.dir/src/lib/cert.c.o -c /home/myMPD/src/lib/cert.c
make[2]: *** No rule to make target '/system/lib/libm.so', needed by 'mympd'.  Stop.
make[2]: Leaving directory '/home/myMPD/debug'
make[1]: *** [CMakeFiles/Makefile2:100: CMakeFiles/mympd.dir/all] Error 2
make[1]: Leaving directory '/home/myMPD/debug'
make: *** [Makefile:136: all] Error 2

What else can I provide to assist you in identifying my myMPD installation issue?

**Comment by jcorporation** [comment]
I opened an issue for this: #721

**Comment by CastleCaptain** [comment]
Will do but will be a couple of weeks before I can do so.

**Reply by jcorporation** [reply]
Meanwhile I close the ticket. We can reopen it if the issue is not fixed.
